src/scss/Footer.scss

```diff
@@ -0,0 +1,24 @@
+.footer {
+  background-color: #00233b;
```

Comment:
we will want to consolidate colors into a new `Variables.scss` and give semantic names to them at a later point